app/models/offer.rb

```diff
@@ -215,7 +215,7 @@ def assign_reviewer(reviewer)
   end
 
   def send_new_offer_notification
-    PushService.new.send_notification Channel.reviewer, true, {
```

Comment:
@swatijadhav so earlier this code was sending message to whom? because new offer does not have any reviewer assigned? Just trying to understand.